src/test/java/org/takes/misc/VerboseIterableTest.java

```diff
@@ -0,0 +1,39 @@
+package org.takes.misc;
+
+import java.util.Arrays;
+import java.util.List;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.collection.IsIterableWithSize;
+import org.junit.Test;
+
+/**
+ * Tests for {@link VerboseIterable}.
+ * @author marcus.sanchez (sanchez.marcus@gmail.com)
+ * @version $Id$
+ * @since 0.15.1
+ */
+public class VerboseIterableTest {
+
+    /**
+     * VerboseIterable can return correct size collection.
+     */
+    @Test
+    public void returnsCorrectSize() {
+        final List<String> VALID = Arrays.asList(
```

Comment:
@m4solutions since this variable is not static, it should be lower case. I suggest `valid`. Take a look at http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#variable-names